.env.dev

```diff
@@ -4,6 +4,7 @@ APP_NAME_CRON=Whitebox | Development
 
 POSTGRES_DB_URI=postgresql://postgres:postgres@localhost:5432/postgres
 
-API_KEY=1234567890
 VERSION=0.1.0
-METRICS_CRON=*/15 * * * *
\ No newline at end of file
+METRICS_CRON=*/15 * * * *
+
+MODEL_PATH=src/analytics/data/trained_models
```

Comment:
I think the test folder should be seperate or at least be something like models/test-model so when the tests run locally it can be safely deleted after the tests are done. But whatever you decide.